query_processor/services/content_router.py

```
@@ -0,0 +1,113 @@
+            await ContentRouterService._update_query_request_error(query_request, error_msg)
+            
+            raise Exception(error_msg) from e
+    
+    @staticmethod
+    async def _update_query_request_error(query_request: QueryRequest, error_message: str):
+        try:
+            await sync_to_async(lambda: setattr(query_request, 'status', 'failed'))()
+            await sync_to_async(lambda: setattr(query_request, 'error_message', error_message))()
+            await sync_to_async(query_request.save)()
+            logger.debug(f"Updated QueryRequest {query_request.search_id} with error status")
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)